sdk/cosmos/azure-cosmos-tests/src/test/java/com/azure/cosmos/ProactiveConnectionManagementTest.java

```diff
@@ -123,13 +124,108 @@ public void openConnectionsAndInitCachesWithInvalidCosmosClientConfig(List<Strin
     }
 
     @Test(groups = {"multi-region"}, dataProvider = "proactiveContainerInitConfigs")
-    public void openConnectionsAndInitCachesWithCosmosClient(List<String> preferredRegions, int numProactiveConnectionRegions, int numContainers) {
+    public void openConnectionsAndInitCachesWithContainer(List<String> preferredRegions, int numProactiveConnectionRegions, int ignoredParam3, int ignoredParam4, Duration ignore) {
```

Comment:
what the `ignoredParam3` is used for?

Reply:
The data provider `proactiveContainerInitConfigs` has the following parameters - preferred list of regions, proactive connections region count, no. of containers, min connection pool size per endpoint and aggressive connection establishment duration. So `ignoredParam3` would refer to the no. of containers for this test.